src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRowHandler.java

```diff
@@ -10,16 +11,25 @@
 	protected DiscreteGroundWaterRowMapper rowMap;
 	protected RdbWriter writer;
 	protected int rowNum;
+	protected List<Parameter> parameters;
 
-	public DiscreteGroundWaterRowHandler(RdbWriter writer) {
+	public DiscreteGroundWaterRowHandler(RdbWriter writer, List<Parameter> parameters) {
 		this.writer = writer;
 		rowNum = 0;
 		rowMap = new DiscreteGroundWaterRowMapper();
+		this.parameters = parameters;
+
 	}
 
 	@Override
 	public void processRow(ResultSet rs) throws SQLException {
 		DiscreteGroundWater dgw = rowMap.mapRow(rs, ++rowNum);
+		for (Parameter parameter: parameters) {
+			if (parameter.parameterCode.equals(dgw.parameterCode)) {
+				dgw.aboveDatum = parameter.aboveDatum;
+				dgw.belowLandSurface = parameter.belowLandSurface;
+			}
+		}
```

Comment:
So each parameter has a flag for both above-datum and below-land?

Reply:
All the parameters that know we care about, yes.

![image](https://user-images.githubusercontent.com/22546918/94824504-b9c67400-03ca-11eb-818a-6ee6736a3669.png)

Reply:
Notice the two entries for 72019 have different datum names specified, but they are both mapped as below land surface.